dashboard/app/routes/optimization/fine-tuning/model_options.ts

```diff
@@ -0,0 +1,45 @@
+import { z } from "zod";
+
+export const ModelOptionSchema = z.object({
+  displayName: z.string(),
+  name: z.string(),
+  provider: z.enum(["openai", "fireworks", "mistral"]),
+});
+
+export type ModelOption = z.infer<typeof ModelOptionSchema>;
+
+// TODO: make a type per model provider containing
+// what information is needed in order to start a fine-tuning job.
+
+export const models: ModelOption[] = [
+  {
+    displayName: "gpt-4o",
+    name: "gpt-4o",
+    provider: "openai",
+  },
+  {
+    displayName: "gpt-4o-mini",
+    name: "gpt-4o-mini",
+    provider: "openai",
+  },
+  {
+    displayName: "gpt-4o-large",
+    name: "gpt-4o-large",
+    provider: "openai",
+  },
+  {
+    displayName: "gpt-4o-turbo",
+    name: "gpt-4o-turbo",
+    provider: "openai",
+  },
+  {
+    displayName: "gpt-4o-mini-2024-07-18",
+    name: "gpt-4o-mini-2024-07-18",
+    provider: "openai",
+  },
+  {
+    displayName: "llama-3.1-8b-instruct",
+    name: "accounts/fireworks/models/llama-v3p1-8b-instruct",
+    provider: "fireworks",
+  },
+];
```

Comment:
Put only things we can actually fine-tune here.

Reply:
I cut this down to two for now, but we should come back and add everything possible that OpenAI and fireworks supports